### PR TITLE
Task 7

### PR DESCRIPTION
**Pull request by AverchenkoAA**
Now you have to add `authorization_token` into local storage with encoded value for `averchenkoaa` user to use `/import` endpoint.

### CODE REVIEW

**Comment by scolt** [comment]
What is about alerting?
There is CORS error while trying to access the endpoint without the token.

File: src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx [hunk 1]
```diff
@@ -37,7 +37,8 @@ export default function CSVFileImport({url, title}: CSVFileImportProps) {
         url,
         params: {
           name: encodeURIComponent(file.name)
-        }
+        },
+        headers:{Authorization: localStorage.getItem('authorization_token')}
```
Comment (MaxPlav) [inline src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx:41]: should be: 'Basic ' + token

**Comment by AverchenkoAA** [comment]
> Is there no alerting?
> There is CORS error while trying to access the endpoint without the token.

There is no alerting. FE just ignore all requests without token.

File: src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx (hunk 1 above)
Comment (AverchenkoAA) [reply src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx]: Fixed in the last commit

File: src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx [hunk 2]
```diff
@@ -32,12 +32,14 @@ export default function CSVFileImport({url, title}: CSVFileImportProps) {
...
       // Get the presigned URL
+      const token = localStorage.getItem('authorization_token');
       const response = await axios({
         method: 'GET',
         url,
         params: {
           name: encodeURIComponent(file.name)
-        }
+        },
+        headers:{Authorization: 'Basic ' + token}
```
Comment (kzanton) [inline src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx:42]: You shouldn't send the authorization header when you have no token

Comment (AverchenkoAA) [reply src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx]: Fixed.

**Comment by AverchenkoAA** [comment]
To use page https://dwgz1q98lr4ie.cloudfront.net/admin/products you should authorize in cognito.

**Pull request merged**